### ISSUE TITLE
 com.hazelcast.jet.impl.connector.ReadWithPartitionIteratorP_MigrationDetectionTest.when_migration_then_detected_remote

### ISSUE THREAD

**Issue by cangencer** [hazelcast/hazelcast-jet#1108]
https://hazelcast-l337.ci.cloudbees.com/view/Jet/job/jet/com.hazelcast.jet$hazelcast-jet-core/927/testReport/junit/com.hazelcast.jet.impl.connector/ReadWithPartitionIteratorP_MigrationDetectionTest/when_migration_then_detected_remote/

```
Error Message
Expected test to throw (an instance of java.util.concurrent.ExecutionException and exception with message a string containing "migration detected")
Stacktrace
java.lang.AssertionError: Expected test to throw (an instance of java.util.concurrent.ExecutionException and exception with message a string containing "migration detected")
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.rules.ExpectedException.failDueToMissingException(ExpectedException.java:263)
	at org.junit.rules.ExpectedException.access$200(ExpectedException.java:106)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:245)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
```

**Comment by viliam-durina** [comment]
Fixed in #1104

https://github.com/hazelcast/hazelcast-jet/pull/1104/commits/b389fa0d14de7ae51864aa0c6ac73f408b61f692

**Comment by viliam-durina** [comment]
Also fixed in #1107

**Comment by cangencer** [comment]
http://jenkins.hazelcast.com/view/Jet/job/jet-oss-master-oracle-jdk11/5/testReport/junit/com.hazelcast.jet.impl.connector/ReadWithPartitionIteratorP_MigrationDetectionTest/when_migration_then_detected_remote/

```
Error Message
Expected test to throw exception with message a string containing "migration detected"
Stacktrace
java.lang.AssertionError: Expected test to throw exception with message a string containing "migration detected"
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.rules.ExpectedException.failDueToMissingException(ExpectedException.java:263)
	at org.junit.rules.ExpectedException.access$200(ExpectedException.java:106)
	at org.junit.rules.ExpectedException$ExpectedExceptionStatement.evaluate(ExpectedException.java:245)
	at com.hazelcast.test.DumpBuildInfoOnFailureRule$1.evaluate(DumpBuildInfoOnFailureRule.java:37)
	at com.hazelcast.test.jitter.JitterRule$1.evaluate(JitterRule.java:104)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
```
